docs/pages/klogo.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+
+  <DocsPageTemplate apiDocs>
+    <DocsPageSection title="Overview" anchor="#overview">
+      <p>Shows Kolibri logo.</p>
+
+      <DocsShow>
+        <KLogo
+          altText="Kolibri Logo"
+          :height="150"
+          width="100%"
+          :maxWidth="150"
+        />
+      </DocsShow>
+      <DocsShowCode language="html">
+        <KLogo
+          altText="Kolibri logo"
+          :height="150"
+          width="100%"
+          :maxWidth="150"
+        />
+      </DocsShowCode>
+    </DocsPageSection>
+
+    <DocsPageSection title="Usage" anchor="#usage">
+      <h3>Alternative text</h3>
+
+      <p>Alternative text (<code>altText</code>) is always required for the logo image. When creating it, consider the following:</p>
```

Comment:
Looking good @MisRob, I'd just suggest to remove 'always', as in the wide world logos are not absolutely required to have alternative text. They can also be used for decorative purposes, but I'm unaware we are using ours that way anywhere in Kolibri. In our case just 'required' should suffice'.

Reply:
Alright, will do before merging. Thank you